versions/3.0.md

```diff
@@ -476,42 +477,46 @@ Field Pattern | Type | Description
   "summary": "Updates a pet in the store with form data",
   "description": "",
   "operationId": "updatePetWithForm",
-  "consumes": [
-    "application/x-www-form-urlencoded"
-  ],
-  "produces": [
-    "application/json",
-    "application/xml"
-  ],
   "parameters": [
     {
       "name": "petId",
       "in": "path",
       "description": "ID of pet that needs to be updated",
       "required": true,
       "type": "string"
-    },
-    {
-      "name": "name",
-      "in": "formData",
-      "description": "Updated name of the pet",
-      "required": false,
-      "type": "string"
-    },
-    {
-      "name": "status",
-      "in": "formData",
-      "description": "Updated status of the pet",
-      "required": false,
-      "type": "string"
     }
   ],
+  "requestBody" : {
+    "representations" : {
+      "application/x-www-form-urlencoded":{
+        "schema" : {
+           "properties": {
+              "name" 
+                "description" : "Updated name of the pet",
+                "type": "string"
+              "status":
+                "description" : "Updated status of the pet",
+                "type": "string"
+           },
```

Comment:
It seems like we're going from a flat array of parameters to an object with those fields. Is the intention that we treat the object as an array of form variables?